x-pack/plugins/alerting/server/routes/find_rules.ts

```diff
@@ -111,6 +111,9 @@ const rewriteBodyRes: RewriteResponseCase<FindResult<RuleTypeParams>> = ({
           params,
           connector_type_id: actionTypeId,
         })),
+        ...(snoozeSchedule != null
+          ? { is_snoozed_until: getRuleSnoozeEndTime({ snoozeSchedule, muteAll })?.toISOString() }
```

Comment:
This is where we derive the current snooze end time from the snooze schedule, and handle RRULE parsing on the server side. This could be temporary if we decide to make the `rrule` package available on the frontend in a later PR.